ui/widgets/sortable.js

```diff
@@ -455,15 +483,26 @@ return $.widget( "ui.sortable", $.ui.mouse, {
 				animation = {};
 
 			if ( !axis || axis === "x" ) {
-				animation.left = cur.left - this.offset.parent.left - this.margins.left + ( this.offsetParent[ 0 ] === this.document[ 0 ].body ? 0 : this.offsetParent[ 0 ].scrollLeft );
+				animation.left = cur.left - this.offset.parent.left - this.margins.left +
+					( this.offsetParent[ 0 ] === this.document[ 0 ].body ?
+						0 :
+						this.offsetParent[ 0 ].scrollLeft
+					);
 			}
 			if ( !axis || axis === "y" ) {
-				animation.top = cur.top - this.offset.parent.top - this.margins.top + ( this.offsetParent[ 0 ] === this.document[ 0 ].body ? 0 : this.offsetParent[ 0 ].scrollTop );
+				animation.top = cur.top - this.offset.parent.top - this.margins.top +
+					( this.offsetParent[ 0 ] === this.document[ 0 ].body ?
+						0 :
+						this.offsetParent[ 0 ].scrollTop
+					);
 			}
 			this.reverting = true;
-			$( this.helper ).animate( animation, parseInt( this.options.revert, 10 ) || 500, function() {
-				that._clear( event );
-			} );
+			$( this.helper ).animate(
+					animation,
+					parseInt( this.options.revert, 10 ) || 500,
+					function() {
+						that._clear( event );
+					} );
```

Comment:
This whole block is over-indented. Also move the closing paren to the next line.